src/backends/apt.rs

```diff
@@ -15,15 +15,16 @@ pub struct AptOptions {}
 
 impl Backend for Apt {
     type Options = AptOptions;
+    type Config = ();
```

Comment:
I'm wondering if we should follow the convention of options and have an empty struct even if there are no config items. I think it makes the macros easier as then every backend is consistent in that they have a specific config type.

Reply:
We can use apply_backends to generate the config struct the same way we make the packages struct, possibly also move it into `all.rs` too. Indeed we can just not document them for now the same way we don't document the per package options for backends without any options.